WordPress/Classes/ViewRelated/Post/Scheduling/PublishSettingsViewController.swift

```diff
@@ -156,9 +165,18 @@ private struct DateAndTimeRow: ImmuTableRow {
         let footerText: String?
 
         if let date = viewModel.date {
-            let publishedOnString = date.longStringWithTime()
-            let offsetLabel = viewModel.timeZone?.label ?? NSLocalizedString("Unknown UTC Offset", comment: "Unknown UTC offset label")
-            footerText = String.localizedStringWithFormat("Post will be published on %@ in your site timezone (%@)", publishedOnString, offsetLabel)
+            let publishedOnString = viewModel.dateTimeFormatter.string(from: date)
+
+            let nameStyle: NSTimeZone.NameStyle = viewModel.timeZone.isDaylightSavingTime(for: date) ? .daylightSaving : .standard
```

Comment:
The display format of the timezone has changed. As originally designed and implemented:

<img width="406" alt="Screen Shot 2019-12-17 at 1 21 16 PM" src="https://user-images.githubusercontent.com/1816888/71031863-7c118c80-20d1-11ea-92e3-74a53ec557fa.png">

If we revert back to that display, checking `isDaylightSavingTime` should not be necessary.

Reply:
I thought this format may match the "Site Timezone" setting a little better. i.e. if you set to `UTC-7` it will reflect that as `GMT-07:00` but if you use `Chicago` it will properly reflect that the timezone is in use and would use whatever rules apply to that zone.